java/src/org/openqa/selenium/grid/config/TomlConfig.java

```diff
@@ -114,6 +132,15 @@ public Set<String> getSectionNames() {
     return ImmutableSortedSet.copyOf(toml.keySet());
   }
 
+  public String getTableAsString(TomlTable table) {
```

Comment:
Maybe we can do something like this ?

```java
return map.entrySet().stream()
.map(e -> e.getKey() + "=" + e.getValue())
.collect(Collectors.joining(" "));
```

Reply:
Doesn't matter because the tests fail due to difference in the parsing functionality of the library 

![image](https://github.com/user-attachments/assets/08749774-37d8-49d2-8f11-b1c36e615b89)